api.yaml

```diff
@@ -322,13 +322,13 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
-  /v1/cloud/arnid/{arnid}:
+  /v1/cloud/arnid/{resourceid}:
     get:
       summary: "Retrieve a cloud asset at a point in time by ARN ID"
```

Comment:
nit: update from `ARN ID` to `Resource ID`

Reply:
do you mean the comment here? I can change that

Reply:
I'll do this in a separate PR

Reply:
yep, text in summary